test/integration/freetext_editor_spec.js

```diff
@@ -252,4 +252,176 @@ describe("Editor", () => {
+        }
+        results.sort();
+        return results;
+      }, editorPrefix.slice(1));
+    }
+
+    it("must select/unselect several editors and check copy, paste and delete operations", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          await page.click("#editorFreeText");
+
+          const rect = await page.$eval(".annotationEditorLayer", el => {
+            // With Chrome something is wrong when serializing a DomRect,
+            // hence we extract the values and just return them.
+            const { x, y } = el.getBoundingClientRect();
+            return { x, y };
+          });
+
+          const editorCenters = [];
+          for (let i = 0; i < 4; i++) {
+            const data = `FreeText ${i}`;
+            await page.mouse.click(
+              rect.x + (i + 1) * 100,
+              rect.y + (i + 1) * 100
+            );
+            await page.type(`${editorPrefix}${i} .internal`, data);
+
+            const editorRect = await page.$eval(`${editorPrefix}${i}`, el => {
+              const { x, y, width, height } = el.getBoundingClientRect();
+              return {
+                x,
+                y,
+                width,
+                height,
+              };
+            });
+            editorCenters.push({
+              x: editorRect.x + editorRect.width / 2,
+              y: editorRect.y + editorRect.height / 2,
+            });
+
+            // Commit.
+            await page.mouse.click(
+              editorRect.x,
+              editorRect.y + 2 * editorRect.height
+            );
+          }
+
+          await page.keyboard.down("Control");
+          await page.keyboard.press("a");
+          await page.keyboard.up("Control");
+
+          expect(await getSelected(page))
+            .withContext(`In ${browserName}`)
+            .toEqual([0, 1, 2, 3]);
+
+          await page.keyboard.down("Control");
+          await page.mouse.click(editorCenters[1].x, editorCenters[1].y);
+
+          expect(await getSelected(page))
+            .withContext(`In ${browserName}`)
+            .toEqual([0, 2, 3]);
+
+          await page.mouse.click(editorCenters[2].x, editorCenters[2].y);
+
+          expect(await getSelected(page))
+            .withContext(`In ${browserName}`)
+            .toEqual([0, 3]);
+
+          await page.mouse.click(editorCenters[1].x, editorCenters[1].y);
+          await page.keyboard.up("Control");
+
+          expect(await getSelected(page))
+            .withContext(`In ${browserName}`)
+            .toEqual([0, 1, 3]);
+
+          await page.keyboard.down("Control");
+          await page.keyboard.press("c");
+          await page.keyboard.up("Control");
+
+          await page.keyboard.down("Control");
+          await page.keyboard.press("v");
+          await page.keyboard.up("Control");
+
+          // 0,1,3 are unselected and new paster editors are selected.
```

Comment:
Nit: Did you mean the following?
```suggestion
          // 0,1,3 are unselected and newly pasted editors are selected.
```